docs/examples/data_connectors/PsychicDemo.ipynb

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
very nit: would prefer if you keep the outputs in the notebook.

Reply:
Could you also link this into our documentation via `docs/how_to/data_connectors.md` See bottom of the page for adding to the `Examples` toctree.

Reply:
Will do on both counts! re: outputs that's just a matter of not using the logger and not using `display()`?